test/utilities/miscellaneous.ts

```diff
@@ -43,3 +42,65 @@ export function currentOsUserName(): string {
 export function transformedUserName(): string {
   return currentOsUserName().replace('.', '_');
 }
+
+/**
+ * @param type type of html tag we want to find
+ * @param attribute attribute that holds the given text
+ * @param labelText text of the element we want to find
+ * @returns element that contains the given text
+ */
+export async function findElementByText(
+  type: string,
+  attribute: string,
+  labelText: string
+): Promise<WebdriverIO.Element> {
+  let element = await $(`${type}[${attribute}="${labelText}"]`);
+  return element!;
+}
+
+/**
+ * @param operation identifies if it's a pull or push operation
+ * @param changes indicates if changes are expected or not
+ * @param type indicates if the metadata is expected to have been created, changed or deleted
+ * @returns the output panel text after
+ */
+export async function verifyPushAndPullOutputText(
+  operation: string,
+  type?: string
+): Promise<string | undefined> {
+  // Check the output.
+  const outputPanelText = await attemptToFindOutputPanelText(
+    `Salesforce CLI`,
+    `=== ${operation}ed Source`,
+    10
+  );
+  expect(outputPanelText).not.toBeUndefined();
+  // expect(outputPanelText).toContain('ended with exit code 0');
+  if (type) {
+    if (operation === 'Push') {
+      expect(outputPanelText).toContain(`${type}  ExampleApexClass1  ApexClass`);
+    } else {
+      expect(outputPanelText).toContain(`${type}  Admin`);
+      expect(outputPanelText).toContain('ended with exit code 0');
```

Comment:
This line is present in all 3 of the cases that you have listed in the helper function.  Can you move this to right before the return?
![Screenshot 2023-11-16 at 9 50 07 AM](https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/139700604/63e0511f-6823-4f80-9b87-ffff8e4ee451)
![Screenshot 2023-11-16 at 9 51 13 AM](https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/139700604/6f3145f6-a9e5-4ca1-8fdd-c44dc100e737)